web/packages/teleport/src/Audit/EventList/EventTypeCell.tsx

```diff
@@ -200,6 +200,13 @@ const EventIconMap: Record<EventCode, React.FC> = {
   [eventCodes.INSTANCE_JOIN]: Icons.Info,
   [eventCodes.LOGIN_RULE_CREATE]: Icons.Info,
   [eventCodes.LOGIN_RULE_DELETE]: Icons.Info,
+  [eventCodes.SAML_IDP_AUTH_ATTEMPT]: Icons.Info,
+  [eventCodes.SAML_IDP_SERVICE_PROVIDER_CREATE]: Icons.Info,
+  [eventCodes.SAML_IDP_SERVICE_PROVIDER_CREATE_FAILURE]: Icons.Info,
```

Comment:
Seems odd to use the same "info" icon for failures, but looks liek we do that all over the place.

Reply:
Yeah, I was hoping for a better icon there but found the same thing.